source/source/deengames/planetoid/entity/Player.hx

```diff
@@ -16,7 +16,7 @@ class Player
     {
         return new Entity()
             .add(new ColourComponent(255, 0, 0, 32, 32))
-            //.add(new SpriteComponent('assets/images/butterfly.png'))
```

Comment:
My bad, I didn't see both your changes when I added that. You can ignore this comment.